lms/djangoapps/discussion_api/api.py

```diff
@@ -629,8 +652,12 @@ def update_thread(request, thread_id, update_data):
     _check_editable_fields(cc_thread, update_data, context)
     serializer = ThreadSerializer(cc_thread, data=update_data, partial=True, context=context)
     actions_form = ThreadActionsForm(update_data)
-    if not (serializer.is_valid() and actions_form.is_valid()):
-        raise ValidationError(dict(serializer.errors.items() + actions_form.errors.items()))
+    if not actions_form.is_valid():
```

Comment:
Validation was separated because a form needs to have is_valid() called on it before I can access its cleaned_data